env/src/lib.rs

```diff
@@ -337,7 +356,12 @@ impl Log for Logger {
             }
         }
 
-        let _ = writeln!(&mut io::stderr(), "{}", (self.format)(record));
+        match self.target {
+            LogTarget::Stdout => println!("{}", (self.format)(record)),
```

Comment:
Should this be a `writeln!` to `io::stdout` instead? There are some differences around capture in the test harness. Not sure which approach would be more expected.